content/en/docs/reference/config/telemetry/metrics/index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Istio Standard Metrics
+description: Istio standard metrics exported by Istio telemetry.
+weight: 50
+---
+
+The following are the standard service level metrics exported by Istio.
+Istio standard metrics are directly exported by the Envoy proxy since Istio 1.5.
+In prior Istio releases Mixer produced these metrics.
+
+## Metrics
+
+For HTTP, HTTP/2, and GRPC traffic, Istio generates the following metrics:
+
+*   **Request Count** (`istio_requests_total`): This is a `COUNTER` incremented for every request handled by an Istio proxy.
+
+*   **Request Duration** (`istio_request_duration_seconds`): This is a `DISTRIBUTION` which measures the duration of requests.
+
+*   **Request Size** (`istio_request_bytes`): This is a `DISTRIBUTION` which measures HTTP request body sizes.
+
+*   **Response Size** (`istio_response_bytes`): This is a `DISTRIBUTION` which measures HTTP response body sizes.
+
+For TCP traffic, Istio generates the following metrics:
+
+*   **Tcp Byte Sent** (`istio_tcp_sent_bytes_total`): This is a `COUNTER` which measures the size of total bytes sent during response in case of a TCP
+    connection.
+
+*   **Tcp Byte Received** (`istio_tcp_received_bytes_total`): This is a `COUNTER` which measures the size of total
+    bytes received during request in case of a TCP connection.
+
+*   **Tcp Connections Opened** (`istio_tcp_connections_opened_total`): This is a `COUNTER` incremented for every opened connection.
+
+*   **Tcp Connections Closed** (`istio_tcp_connections_closed_total`): This is a `COUNTER` incremented for every closed connection.
+
+## Labels
```

Comment:
we are missing `destination|source_canonical_service` and `destination|source_canonical_revision`